docs/iris/src/whatsnew/contributions_2.0/incompatiblechange_2017-Aug-29_fill_value_save_arg.txt

```diff
@@ -0,0 +1 @@
+* When saving a cube or list of cubes, a fill value or list of fill values can be specified via a new `fill_value` argument. If a list is supplied, each fill value will be applied to each cube in turn. If a `fill_value` argument is not specified, the default fill value for the file format and the cube's data type will be used. Fill values are no longer taken from the cube's `data` attribute when it is a masked array.
```

Comment:
This isn't a general facility -- it only applies to certain save formats (netcdf only here, PP to be added shortly..)

Reply:
ok, I'll add mentions for both file formats here.